### DISCUSSION TITLE
a lot of errors in event viewer

### DISCUSSION THREAD

**Discussion by RobaloAmigo** [builtbybel/Bloatynosy#118] [General]
After using it (even carefully and turning off only few most irritating Win11 "feaures") there's A LOT of errors in event viewer. Partially it can be fixed by i.e. turning off DeviceSetupManager or LSA logs in registry, but this is something that this app should do automatically while now it makes event viewer look like trash. It's not enough comprehency at this moment in my opinion

**Comment by RobaloAmigo** [comment]
another issue - this app stopped Task Scheduler to run so a lot of apps like MSI Afterburner don't work properly. seems not tested crap.

**Comment by LinuxOnTheDesktop** [comment]
If I may chime in: I was about to commend this (wonderfully named!) application to someone; but the comments above (intemperate as the second of them is) gave me pause.

**Reply by RobaloAmigo** [reply]
I'm glad someone was saved.
I personally had to reinstall Windows to get rid of all the issues generated by this software.
